app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  class BaseController < ApplicationController
+    respond_to :json
+
+    SUCCESS = 'SUCCESS'.freeze
+    ERROR = 'ERROR'.freeze
```

Comment:
I could go either way, but: Maybe we don't need these? Seems like a decent amount of effort and trust in the developer to remember to include them in every response, and they're largely redundant with the status code.